configuration_example-d2-docker.json

```diff
@@ -17,6 +18,65 @@
         "username": "system",
         "password": "System123"
     },
+    "departments":
+    {
+      "DEP1": {
+        "datasets": [
+          "example_uid", "example_uid2"
+        ]
+      },
+      "DEP2": {
+        "eventprograms": [
+          "example_uid"
+        ],
+        "datasets": [
+          "example_uid"
+        ]
+      },
+      "DEP3": {
+        "eventprograms": [
+          "example_uid"
+        ],
+        "datasets": [
+          "example_uid"
+        ],
+        "trackerprograms": [
```

Comment:
We use camel case throughout the code, so maybe `eventprograms` -> `eventPrograms`, `trackerprograms` -> `trackerPrograms` and so on.